api/proto/common/common.proto

```diff
@@ -9,24 +9,13 @@ message G1Commitment {
 	bytes y = 2;
 }
 
-message G2Commitment {
```

Comment:
shouldn't be used other than in `BlobCommitment` in this file, which isn't in production use